### PR TITLE
mrtydi test (for untied-nq)

### PR DESCRIPTION
**Pull request by crystina-z**
Add test case for mrtydi dense results. as in slack discussion

### CODE REVIEW

File: integrations/dense/test_mrtydi.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                                --threads {self.threads} \
+                                --encoder castorini/mdpr-question-nq'
+            cmd2 = f'python -m pyserini.eval.trec_eval -c -M 100 -m recip_rank -c -m recall.100 mrtydi-v1.1-{lang}-test {output_file}'
+ 
+            status1 = os.system(cmd1)
+            stdout, stderr = run_command(cmd2)
+            score = parse_score(stdout, 'recip_rank')
+ 
+            self.assertEqual(status1, 0)
+            self.assertAlmostEqual(score, expected_score, delta=0.01)
```
Comment (lintool) [inline integrations/dense/test_mrtydi.py:66]: smaller delta? Start with 0.001.

If it becomes too tight, we can loosen it.

**Review by lintool — CHANGES_REQUESTED** [review]
You want a teardown method as well:

```
    def tearDown(self):
        clean_files(self.temp_files)


if __name__ == '__main__':
    unittest.main()
```

File: integrations/dense/test_mrtydi.py (hunk 1 above)
Comment (crystina-z) [reply integrations/dense/test_mrtydi.py:66]: ah for some language the difference is somehow slightly over 0.005, I'll see what's the tightest bound we can have now

**Comment by crystina-z** [comment]
updated, using delta=0.0012 as the largest gap observed is on telugu; (0.1099 v.s 0.111)

**Comment by lintool** [comment]
> updated, using delta=0.0012 as the largest gap observed is on telugu; (0.1099 v.s 0.111)

I think you're missing a zero?

**Comment by crystina-z** [comment]
oh I meant the previous score was 0.111 on Telugu(after rounding), and now we get 0.1099,  and the delta is slightly over 0.0011 so was setting it as 0.0012 here

**Review by lintool — APPROVED** [review]

**Pull request merged**